webapp/sources/rudder/rudder-web/src/main/webapp/javascript/rudder/homePage.js

```diff
@@ -216,11 +216,6 @@ function doughnutChart (id,data,count,colors) {
                       break;
                   case 'nodeAgents':
                       query.query.where = [{
-                          objectType: "software"
-                        , attribute : "cn"
-                        , comparator: "eq"
-                        , value     : "rudder-agent"
```

Comment:
I'm not sure if this is important, if I follow the logic `Rudder agent (DSC)` whould be use when a version contains a windows node (`rudder-agent OR Rudder agent (DSC)`)

Reply:
the problem is that t will match any software in the same version, which is not that likely to happen but would be very surprising for the user.
Can't we use a regex to match both `rudder-agent` and `Rudder agent (DSC)` in the filter?

Reply:
We should look for the version we have in the "management technology" in out inventory (that's an entry we control and now we set Rudder version within, it was not the case in the old days